doc/ledger.1

```diff
@@ -845,7 +851,10 @@ but show both sides of each transaction.
 .\".It Fl \-revalued
 .\".It Fl \-revalued-only
 .\".It Fl \-revalued-total Ar EXPR
-.\".It Fl \-rich-data
+.It Fl \-rich-data
+When generating ledger transaction from CSV file using the
+.Ic convert
+command, add CSV, Imported, and UUID meta-data.
```

Comment:
I have added an example for --rich-data in the manual, and also I also explained the associated tags. See additional commit.

But I will not add this to man page, that serve a different philosophy compared to texinfo file (see end of section Output_formats of http://en.wikipedia.org/wiki/Texinfo#Output_formats, and also http://www.gnu.org/software/texinfo/manual/texinfo/texinfo.html#Adding-Output-Formats). The man page serve as a reference card, and actually the current man page of ledger does not contain any example and explanation of each option is very short. I do not judge good or bad, but as a matter of fact, Ledger is more of less following same philosophy as Debian, with close to empty man page, referring to info if more is needed.